optoforce/src/optoforce/optoforce.py

```diff
@@ -69,8 +70,13 @@ def __init__(self):
         """
         Initialize OptoforceDriver object
         """
-        port = rospy.get_param("~port", "/dev/ttyACM0")
-        self._serial = serial.Serial(port, 1000000, timeout=None)
+        try:
+            port = rospy.get_param("~port", "/dev/ttyACM0")
+            self._serial = serial.Serial(port, 1000000, timeout=None)
+        except serial.SerialException as e:
+            rospy.logfatal(e.message)
+            rospy.signal_shutdown("Serial connection failure")
```

Comment:
The message that is actually displayed is e.message. It does not prompt the user to check for an other port but clearly states that the node could not open the port (and gives the path). Could that be enough in your opinion ?